lib/modules/manager/sbt/extract.ts

```diff
@@ -289,13 +323,21 @@ const query = q.tree<Ctx>({
   postHandler: registryUrlHandler,
 });
 
+// Extract 1 file
 export function extractPackageFile(
   content: string,
-  packageFile: string
-): PackageFileContent | null {
+  {
+    packageFile,
+    registryUrls,
+    localVars,
+    globalVars,
+    scalaVersion,
+  }: PackageFile & ParseOptions
```

Comment:
Please don't use type intersection, instead add the missing fields to the `ParseOptions`

Reply:
done